packages/web-app-files/src/services/folder/loaderSharedWithOthers.ts

```diff
@@ -51,6 +51,9 @@ export class FolderLoaderSharedWithOthers implements FolderLoader {
         )
       }
 
+      const user = store.state.user
+      resources = resources.filter((r) => r.shareOwner === user.id)
```

Comment:
shared_with_me = false seems not to work, therefore filtering

Reply:
Did you manage to confirm that this happens with oCIS backend only? If yes please create an issue in the ocis repo and add the link here together with a `FIXME` comment.

Reply:
As far as I understood a request to `shares` should only return the outgoing shares, having `shared_with_me=true` flips it around to only incoming shares. But it's also possible that I just got that wrong. 😅

Reply:
Done